examples/projections/table/README.txt

```diff
@@ -0,0 +1,83 @@
+Projection Table
+----------------
+
+PyGMT offers 31 map projections. These are specified using the ``projection`` argument.
+The projection codes for the projection styles are shown below.
```

Comment:
There are similar sentences at the top of this page (https://pygmt-f6khtwuqs.vercel.app/projections/index.html).
```suggestion
The below table shows the projection codes for the 31 GMT projections.
```

Reply:
@willschlitzer I'm not a native English speaker, you can rewrite the sentence if it doesn't read well.

Reply:
Looks good to me!